sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -254,35 +253,47 @@ internal class BitmapWidgetRenderer(
     this.bitmap = bitmap
     this.halfBitmapWidth = bitmap.width / 2f
     this.halfBitmapHeight = bitmap.height / 2f
-    setTranslation(this.translationX, this.translationY)
+    updateTranslateMatrix()
     updateVertexBuffer()
     updateMatrix = true
     needRender = true
   }
 
   override fun setRotation(angleDegrees: Float) {
+    rotation = angleDegrees
     Matrix.setIdentityM(rotationMatrix, 0)
     Matrix.setRotateM(rotationMatrix, 0, angleDegrees, 0f, 0f, 1f)
```

Comment:
it seems that `rotationMatrix` / `translateMatrix` and other stuff should thread-safe. Not sure that's in scope of this PR but setting them happens on main thread while they are used for drawing on render thread

Reply:
We discussed it offline in the context https://github.com/mapbox/mapbox-maps-android/pull/1782#discussion_r1011384953 and decided to do in another PR

Reply:
This also might related to the flickering of widgets as reported earlier, need to be followed up separately.